templates/inspec/examples/google_spanner_database/google_spanner_databases.erb

```diff
@@ -0,0 +1,6 @@
+<% gcp_project_id = "#{external_attribute('gcp_project_id', doc_generation)}" -%>
+<% spannerdatabase = grab_attributes['spannerdatabase'] -%>
+
+describe google_spanner_databases(project: <%= gcp_project_id -%>, , instance: <%= doc_generation ? "'#{spannerdatabase['instance']}'" : "spannerdatabase['instance']" -%>) do
+  its('instance') { should eq <%= doc_generation ? "'#{spannerdatabase['instance']}'" : "spannerdatabase['instance']" -%> }
```

Comment:
This is a test for a plural resource, which doesn't have a field called `instance`. It should have `instances` which contains an array of the values from the `instance` field, so this could look like

`its('instances') { should include ... }`

Reply:
done